src/DotVVM.Framework/Routing/RedirectingPresenter.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DotVVM.Framework.Hosting;
+
+namespace DotVVM.Framework.Routing
+{
+    public class RedirectingPresenter : IDotvvmPresenter
```

Comment:
I understand the presenter is used for redirection, but it actually does not do any redirect on itself. Let's not bikeshed about a better name and make the class internal instead.